src/main/java/core/basesyntax/dao/ConverterDataToObjImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.FruitTransaction;
+import java.util.List;
+
+public class ConverterDataToObjImpl {
+    private static final String SEPARATOR = ",";
+    private static final int OPERATION_SEQUENCE = 0;
+    private static final int FRUIT_NAME_SEQUENCE = 1;
+    private static final int FRUIT_QUANTITY_SEQUENCE = 2;
+
+    public List<FruitTransaction> dataConverter(List<String> listOfStrings) {
```

Comment:
Method names are verbs, variables and classes are nouns. When you do otherwise codes becomes much harder to be understood
```suggestion
    public List<FruitTransaction> convertAll(List<String> listOfStrings) {
```